ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -484,11 +454,11 @@ public void writeIndexToFile(long safeEndPoint)
         // write key, value size and file end pointer for this index
         writer.writeInt(this.keySize);
         writer.writeInt(this.valueSize);
-        writer.writeLong(safeEndPoint);
```

Comment:
Not sure what you mean. That version _is_ the version for the `IndexSegment`. There cannot be a concept of "version" for the `PersistentIndex` right?